[MI-3812]: Added the test cases for UI of list of linked channels in RHS on MM. (#20)
Brightscout/mattermost-test-management: data/test-cases/plugins/ms-teams-sync/ui-integration/Tool_tip_linked_channel_list.md

---
# (Required) Ensure all values are filled up

> name: "Tool tip upon hovering over the name of linked channel"

Reviewer: ```suggestion
name: "Tooltip on hovering over the linked channel details"
```

Reviewer: tooltip is a single word. fix everywhere

status: Active
priority: Normal
folder: ui-integration
authors: "@arush-vashishtha"
team_ownership: []
priority_p1_to_p4: P2 - Core Functions (Do core functions work?)

# (Optional)
location: null
component: null
tags: []
labels: []
tested_by_contributor: null

# (Optional) Test type and tools
cypress: null
detox: null
mmctl: null
playwright: null
rainforest: []
manual_test_environments: []

# Do not change
id: null
key: null
created_on: null
last_updated: null
case_hashed: null
steps_hashed: null
---

**Step 1**

> 1. Hover over the channel name in the `list of linked channels` in the RHS of MS Teams sync plugin on MM.

Reviewer: Should we mention the team name as well?
